Authentication/AuthenticationBootstrapper.swift

```diff
@@ -63,10 +64,10 @@ public class AuthenticationBootstrapper<User: UserType, SessionService: SessionS
         }
     }
 
-    /**
+    /*
         Bootstraps your project with the authentication framework,
```

Comment:
The only thing  is that then it will start with login or signup according to what the user decides. `authentication process`? Or maybe explaining exactly that: `starting with the login or signup screen (according to you preferences) if no user is logged in.`